### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[1] == command[2]:
+        pass
+    with open(command[1], "r") as file_from:
+        with open(command[2], "w") as file_to:
+            file_to.write(file_from.read())
```
Comment (YukihiroSM) [inline app/main.py:7]: use list unpacking instead of indexes. 
Check, that command is `cp`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,10 @@
...
+
+
+print(copy_file("cp file.txt new_file.txt"))
```
Comment (YukihiroSM) [inline app/main.py:10]: remove this

File: app/main.py [hunk 3]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[1] == command[2]:
+        pass
```
Comment (YukihiroSM) [inline app/main.py:4]: This will not stop the function execution. This if does nothing

File: app/new_file.txt
Comment (YukihiroSM) [inline app/new_file.txt:1]: Remove test files from git. Only `py` file should be in git

File: app/file.txt
Comment (YukihiroSM) [inline app/file.txt:1]: Remove test files from git. Only py file should be in git

**Review by YukihiroSM — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_from, file_to = command.split()
```
Comment (YukihiroSM) [inline app/main.py:2]: check, that command is cp

File: app/main.py [hunk 5]
```diff
@@ -1 +1,7 @@
...
+    if file_from == file_to:
+        return
+    with open(file_from, "r") as file_from:
+        with open(file_to, "w") as file_to:
```
Comment (YukihiroSM) [inline app/main.py:6]: you can do it using one `with`
```suggestion
    with (open(file_from, "r") as file_in,
             open(file_to, "w") as file_out):
```

**Review by YukihiroSM — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 6]
```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_from, file_to = command.split()
```
Comment (olena-hul) [inline app/main.py:2]: Check command length before unpacking. It should be equal exactly to 3 parts

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 7]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command) != 3:
```
Comment (olena-hul) [inline app/main.py:2]: 1 - split
2 - validate length of splitted command
3 - unpack

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 7 above)
Comment (mwellick) [reply app/main.py:2]: oops,sorry

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**